win32ss/gdi/ntgdi/text.c

```diff
@@ -3,7 +3,11 @@
  * LICENSE:         GPL - See COPYING in the top level directory
  * FILE:            win32ss/gdi/ntgdi/text.c
  * PURPOSE:         Text/Font
- * PROGRAMMER:
+ * PROGRAMMERS:     Amine Khaldi <amine.khaldi@reactos.org>
+ *                  Hermes Belusca-Maito <hermes.belusca@sfr.fr>
+ *                  Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
Does this list is sorted alphabetically? If so, James should be above you.

Reply:
Usually no; this is added chronologically.

Reply:
Sorted the authors chronologically.